coincurve/__init__.py

```diff
@@ -1,6 +1,41 @@
-from coincurve.context import GLOBAL_CONTEXT, Context
-from coincurve.keys import PrivateKey, PublicKey, PublicKeyXOnly
-from coincurve.utils import verify_signature
+def load_secp256k1_conda_library():
+    """Load the secp256k1 library from the conda environment."""
+    import warnings
```

Comment:
Let's move this inside the conditions

Reply:
I don't understand what you mean

Reply:
Oh right got it, I was not clear of debug  ... I am a messy eater :(